icu4j/main/charset/pom.xml

```diff
@@ -22,7 +22,7 @@
   <dependencies>
     <dependency>
       <groupId>com.ibm.icu</groupId>
-      <artifactId>core</artifactId>
+      <artifactId>icu4j</artifactId>
```

Comment:
Is this right in the charset pom?

Reply:
It is the correct one.
This is for publishing, and we publish `charset`, `icu4j`, and `localespi`.
We don't publish `core`.
So `charset` depends on `icu4j`, `localespi` depends on both.

You can also see this in `<icu_root>\icu4j\maven\icu4j-charset\pom.xml` at the very end, in the `<dependencies>` section.

---

It would be nice to unify things and publish what we build, instead of bending backwards to create a "fake" `icu4j` artifact.

Reply:
This looks good. I like that this PR focuses on preserving existing behavior, as a first step.

In the future, as a next step, we can consider publishing more granular (ex: component based) artifacts, as Yoshito has been saying.

Reply:
> This looks good. I like that this PR focuses on preserving existing behavior, as a first step.
> 
> In the future, as a next step, we can consider publishing more granular (ex: component based) artifacts, as Yoshito has been saying.

@echeran Ticket please?